snapcraft/yaml.py

```diff
@@ -129,6 +129,8 @@ def __init__(self):
 
             if 'after' in properties:
                 after_requests[part_name] = properties.pop('after')
+                if isinstance(after_requests[part_name], str):
+                    after_requests[part_name] = [after_requests[part_name]]
```

Comment:
This is a little unreadable. You are replacing the var with a list of itself.
I would either add a comment or try to a make it clearer what's going on. What do you think of this?
after_requests[part_name] = []
after_requests.extend(properties.pop('after'))